python/tests/test_regression.py

```diff
@@ -0,0 +1,62 @@
+from vw_executor.vw import Vw
+from vw_executor.vw_opts import Grid
+from numpy.testing import assert_allclose
+import pandas as pd
+import numpy as np
+import random
+
+with open("regression1.txt", "w") as f:
+    for i in range(10000):
+        x = random.uniform(1, 100)
```

Comment:
it is not great to have true randomness in tests. We should seed it before every call